packages/dev/core/src/Meshes/thinInstanceMesh.ts

```diff
@@ -11,6 +11,15 @@ declare module "./mesh" {
          * Gets or sets a boolean defining if we want picking to pick thin instances as well
          */
         thinInstanceEnablePicking: boolean;
+
+        /**
+         * Indicates that a buffer created as static should be recreated if the buffer is updated (by calling thinInstanceSetMatrixAt or thinInstanceSetAttributeAt, for eg.)
+         * If this flag is false, a buffer created as "static" won't show any update done to it, and will stay the same as it was created.

```

Comment:
```suggestion
         * If this flag is false (the default behavior), a buffer created as "static" won't show any update done to it, and will stay the same as it was created.
```

Nit, feel free to ignore that